components/win-users/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "habitat_win_users"
+version = "0.0.0"
+authors = ["Matt Wrock <matt@mattwrock.com>"]
+description = "Habitat library for win32 account api calls"
+build = "../build.rs"
```

Comment:
Yeah, I started with FFI, but was fighting it too long and just needed to ship the PR.